Microsoft.DotNet.Interactive.PowerShell/Commands/NewPlotlyTraceCommand.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using static XPlot.Plotly.Graph;
+
+namespace Microsoft.DotNet.Interactive.PowerShell.Commands
+{
+    using System.Management.Automation;
+
+    /// <summary>
+    /// Takes the the string input and turns it into an IHtmlContent that can be passed in to
+    /// Show-JupyterContent to render Html in a Jupyter cell's output.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "PlotlyTrace")]
+    [OutputType("XPlot.Plotly.Graph+Trace")]
+    public sealed class NewPlotlyTraceCommand : PSCmdlet
+    {
+        /// <summary>
+        /// The object from pipeline.
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 0)]
+        public string TraceType { get; set; }
+
+        [Parameter(Position = 1)]
+        public string Name { get; set; } = "Untitled";
```

Comment:
Same as above whether empty string makes more sense

Reply:
Now set to empty string.